config/samples/sample-app-agent.yml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "3"
+    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"quarkus-test-agent:0.0.12"},"fieldPath":"spec.template.spec.containers[?(@.name==\"quarkus-test-agent\")].image"}]'
```

Comment:
We should leave out these k8s annotations  (same for Service) as they are set by the k8s controller?